gegelatilib/include/learn/adversarialJob.h

```diff
@@ -59,6 +59,16 @@ namespace Learn {
          */
         std::vector<const TPG::TPGVertex*> roots;
 
+        /**
+         * Position of the root that is to be evaluated.
+         * Equal to -1 if there is no one.
+         * That's typically useful if we have e.g. 4 champions
+         * and a root to evaluate in the job : we can
+         * skip the champions and just look at the score of
+         * the root.
```

Comment:
To clarify a bit this comment, it'd be nice to add a pointer to someplace in the code where what this positions means is explained. If there is no such place in the code, there should be one added :D